src/item.cpp

```diff
@@ -3707,7 +3707,41 @@ bool item::flammable() const
     for( auto mat : made_of_types() ) {
         flammability += mat->fire_resist();
     }
-    return flammability <= 0;
+
+    if( flammability == 0 ) {
+        return true;
+    }
+
+    if( made_of("nomex") ) {
+        return false;
+    }
+
+    if( made_of("paper") || made_of("powder") || made_of("plastic") ||
+        type->id == "whiskey" || type->id == "vodka" ||
+        type->id == "rum" || type->id == "tequila" ||
+        type->id == "single_malt_whiskey" || type->id == "gin" ||
+        type->id == "moonshine" || type->id == "brandy") {
```

Comment:
Could we maybe just make a material for these hard-coded types along the lines of "flammable_alcohol"?